.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+end_of_line = lf
+
+[*.{html,js}]
```

Comment:
I think that this particular subset of rules can be removed given the above rules for `[*]`.